CMakeLists.txt

```diff
@@ -198,8 +198,14 @@ elseif (USE_NEKRS)
 endif ()
 
 if (SCALE_FOUND)
-  target_compile_definitions(libenrico PRIVATE USE_SHIFT)
-  list(APPEND LIBRARIES ${SCALE_LIBRARIES} ${SCALE_TPL_LIBRARIES})
+    target_compile_definitions(libenrico PRIVATE USE_SHIFT)
+    list(APPEND LIBRARIES ${SCALE_LIBRARIES} ${SCALE_TPL_LIBRARIES})
+endif ()
+
+# Discover OpenMP.  Needed for CoupledDriver to discover number of threads
+find_package(OpenMP)
+if (OPENMP_CXX_FOUND)
+    list(APPEND LIBRARIES OpenMP::OpenMP_CXX)
```

Comment:
It looks like the namespaced targets like `OpenMP::OpenMP_CXX` were added in CMake 3.9. If we're going to keep this, you should updated the minimum cmake required.

Reply:
Fixed in 79279185de09ba020b25b22a1e289a73aae030f1